.buildkite/scripts/steps/gh_labels.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { execSync } from 'child_process';
+
+const parseTarget = process.argv[2] ?? 'BUILDKITE_MESSAGE';
+
+// @ts-expect-error
+const firstMatch = (x: string) => x[0].match(prNumWithinMsgRe)[1];
+const prNumWithinMsgRe = /\(\#(\d+)\)/;
+const fetchLabels = (prNumber: string | number) =>
+  execSync(
```

Comment:
We'll run into rate limit issues using this unauthenticated.  Something similar to https://github.com/elastic/kibana/blob/main/.buildkite/scripts/steps/cloud/purge_deployments.ts#L26 should work.

Reply:
Can we make sure to wrap this in a try block with an error log limited to the message body too?